### ISSUE TITLE
Add styles to wrapper div

### ISSUE THREAD

**Issue by hendy001** [nygardk/react-break#3]
Currently cannot find a way to style the wrapper div inserted by by react-break. 
The issue I have is that the parent and child div of the wrapper div inserted by react-break have their width set to 100%. However the child width does not extend to the width of the parent, only as wide as the content of the wrapper div. With testing I figured that setting the width of the wrapper to 100% fixed my issue, however there is not way to do this from my code without manually finding the wrapper and changing its properties after render. I figure a better solution is to pass the styles down to the wrapper div from its parent? 

So the resulting code would look something like this

```
import { layoutGenerator } from 'react-break'

const layout = layoutGenerator({
  list: 0,
  grid: 960
})

const OnList = layout.is('list')
const OnGrid = layout.is('grid')

class MyComponent extends React.Component {
  render () {
    return (
      <div style={{ width: '100%' }}>
        <OnList style={{width: '100%'}}>
           <div style={{width: '100%'}}>Full Width</div>
        </OnList>
        <OnGrid>
          <div>Content width</div>
        </OnGrid>
      </div>
    )
  }
}
```

**Comment by nygardk** [comment]
Hi,

If you pass a single element to a layout component, e.g.
```jsx
<OnList>
  <div />
</OnList>
```
react-break it will not create a wrapping div. It is rended just as
```jsx
<div />
```
However, if you pass multiple child elements, e.g.
```jsx
<OnList>
  <div />
  <div />
</OnList>
```
react-break will create a wrapper and the result will be
```jsx
<div className="react-wrapper react-wrapper--is-list">
  <div />
  <div />
</div>
```

I just added an option in react-break@1.1.0 to be able to force creating a wrapper div also for single child elements. E.g.:
```jsx
<OnList forceWrap>
  <div />
</OnList>
```

Note that the react-break classes to wrapper div were also added only in v1.1.0.

Did this help?
